abcd/frontends/commandline/commands.py

```diff
@@ -111,6 +111,9 @@ def summary(*, db, query, print_all, bins, truncate, props, **kwargs):
 
         print('Total number of configurations: {}'.format(total))
 
+        if total == 0:
```

Comment:
I think this entire getting and checking of 'total' should be outside the if props business